### DISCUSSION TITLE
Graphical Glitches in Kirby 64 The Crystal Shards

### DISCUSSION THREAD

**Discussion by Someguy1519** [mupen64plus-ae/mupen64plus-ae#962] [General]
![Screenshot_20211008-203349_M64Plus FZ Pro](https://user-images.githubusercontent.com/27980228/136643062-6420501a-9e78-430d-8929-605f3ddd4e53.jpg)
I get graphical glitches in Kirby 64. The right side of the screen along with the HUD flickers and glitches with the GlideN64 profile.

**Comment by fzurita** [comment]
Is this happening in all levels or just a specific level?

**Comment by fzurita** [comment]
Actually, never mind, I see the problem myself. Thanks for reporting.

**Reply by Someguy1519** [reply]
The Pause Menu is also very glitchy
![Screenshot_20211008-205001_M64Plus FZ Pro](https://user-images.githubusercontent.com/27980228/136643282-d6884be6-0842-4b51-92ae-43bbad4f58b2.jpg)

**Reply by fzurita** [reply]
Yeah, I will have to take a look at this. Not sure what broke it yet.

**Reply by fzurita** [reply]
Upstream gliden64 has a fix for the hud and pause menu, I'm not sure what is causing the flickering borders though. A workaround would be to apply zoom so that the borders are not visible since they are normally black anyways.
